src/pages/patterns/filters.mdx

```diff
@@ -0,0 +1,383 @@
+The number of results after Filtering should be visible to users where possible.
+The number of resulting data items should be displayed above the table below the filtering tags.
+For guidance on formatting loading states, see our [Tables Basic Formatting pattern](/pattern/tables).
+
+## Selection methods
+
+Different components are used depending on the type of selections a user is making to filter their dataset.
+
+| Selection method                                         | Description                                                                                                        | Container                                |
+| -------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ | ---------------------------------------- |
+| Single select                                            | A user wants to use one category or subcategory to filter data                                                     | Menu                                     |
+| Multiple selections, one category or subcategory of data | A user wants to use more than one selection to filter data                                                         | Menu that allows for multiple selections |
+| Multiple selections, few categories of data              | A user wants to use multiple selections within different categories to quickly filter data                         | Nested menu                              |
+| Multiple selections, one category with many options      | A user wants to be able to search options and use multiple selections within a large set of options to filter data | Tooltip modal                            |
+| Many categories, live updates                            | A user wants to set multiple filters across a dataset with many categories and see their dataset update live       | Sheet                                    |
+| Many categories, batch updates                           | A user wants to set multiple filters across a dataset with many categories and apply them all at once              | Drawer                                   |
+
+## Usage
+
+### Basic filtering
+
+Basic Filters allow users to filter a dataset through one category with only a few options.
+
+#### Single select
+
+Single select basic filters use one selection to filter data. Upon the selection,
+the Menu closes and the filter is surfaced as a Tag.
+Selecting another menu option replaces the current Filter with the new selection.
+
+![Filter pattern basic single select 1](figma:filter-usage-basic-singleselect-1.png)
+![Filter pattern basic single select 2](figma:filter-usage-basic-singleselect-2.png)
+![Filter pattern basic single select 3](figma:filter-usage-basic-singleselect-3.png)
+![Filter pattern basic single select 4](figma:filter-usage-basic-singleselect-4.png)
```

Comment:
Wondering if there's another way we can display these images on the page. Right now, the images are stacked on top of each other (without spaces between), which also pushes content down. Perhaps we could cut off some of the bottom of the table to reduce vertical space? Not sure the best approach, just flagging for intrigue.

![Uploading Screenshot 2023-08-08 at 9.09.08 AM.png…]()

I want comment on every single one of the cases but it happens a few times on this page. Might need a good brainstorm.

Reply:
Looks like you may have trimmed the height on a few, but several are still quite tall, including the images I commented on here originally. 🪜 👷🏻